rmgpy/rmg/input.py

```diff
@@ -1096,7 +1096,8 @@ def read_input_file(path, rmg0):
 
     # convert keys from species names into species objects.
     for reactionSystem in rmg.reaction_systems:
-        reactionSystem.convert_initial_keys_to_species_objects(species_dict)
+        if not isinstance(reactionSystem, Reactor):
```

Comment:
In what case a reactionSystem won't be a `Reactor` again? Are we keeping the old base.py objects? Also could you change reactionSystem to reaction_system, thanks!

Reply:
Exactly, for now we're keeping the old base.pyx objects which won't be Reactor objects

Reply:
fixed